### ISSUE TITLE
Ergänzung der ORCID in 3000/3010 "$p"

### ISSUE THREAD

**Issue by socheres** [ubtue/DatenProbleme#2130]
### Problembeschreibung
die aus den Webseiten entnommene ORCID in 8910 wird im Unterfeld "$p" 3000/3010 ergänzt. Ausgenommen davon sind 3000/3010 mit einer PPN-Verknüpfung.

Beispiel:

3000 Kelley, Michelle L.$porcid:0000-0003-3643-0419$BVerfasserIn$4aut
3010 Strowger, Megan$porcid:0000-0002-8476-5099$BVerfasserIn$4aut
3010 !077404084!$BVerfasserIn$4aut

8910 $aixzom$borcid:0000-0002-8476-5099 | Megan Strowger
8910 $aixzom$borcid:0000-0003-3643-0419 | Michelle L. Kelley
8910 $aixzom$bAutor in der Zoterovorlage [Zenger, Erich] maschinell zugeordnet

**Comment by IxKo** [comment]
> Per Mail um Struktur-Anpassung gebeten (Leerzeichen nach Doppelpunkt): $porcid: 0123-0123-0123-0123

Danke, wurde umgesetzt!

**Comment by socheres** [comment]
@IxKo der Code hat sich noch geändert, bitte noch einmal testen. Vielen Dank!

**Comment by IxKo** [comment]
Nun funktioniert es nicht mehr korrekt:
Bei einem Zweitautor ist eine ORCID, die aber auch beim ersten Autor übertragen wird:
![image](https://github.com/ubtue/DatenProbleme/assets/57000004/307249c0-c3e1-497b-b117-b6334811ba50)

Auch bei zwei Autoren und zwei ORICDs wird bei beiden die gleiche ORICD in 30XX übertragen:
![image](https://github.com/ubtue/DatenProbleme/assets/57000004/52aa432d-bed3-4b94-818a-d2888cd435f8)

**Comment by socheres** [comment]
Danke! Ich habe die letzte Änderung rückgängig gemacht.

**Comment by socheres** [comment]
>Das ist wahrscheinlich zu vernachlässigen, da es zusätzliche Information ist.

Ja, das ist mir auch aufgefallen. Vielleicht könnte man diese Information später zur Verbesserung des Nomrdatensatzes verwenden. Danke!

**Comment by socheres** [comment]
@IxKo würden Sie erneut testen. Bei der PPN-Verknüpfung sieht der Kommentar in 8910 jetzt anders aus. Anstelle der ORCID steht jetzt eine PPN, was logischer ist.

**Comment by IxKo** [comment]
Das fände ich auch logischer, doch bei mir hat sich nichts geändert und es wird noch immer $porcid: ... angestellt.
Ich habe Zotero mehrfach neugestartet.

**Comment by socheres** [comment]
Sollten Sie noch weitere Verbesserungsvorschläge haben, kann ich sie gerne umsetzen.

**Comment by IxKo** [comment]
Ich bin eben auf einem Fall gestoßen, bei dem es sich anders verhält:
https://doi.org/10.25365/exf-2022-1-6
![image](https://github.com/ubtue/DatenProbleme/assets/57000004/a3afb846-a08a-4e9b-8449-9e3da1c6e4fc)

**Comment by socheres** [comment]
Dies ist der Fall, wenn der Normdatensatz auch eine ORCID enthält:https://swb.bsz-bw.de/DB=2.104//CMD?RETRACE=0&TRM_OLD=&ACT=SRCHA&IKT=2072&SRT=RLV&TRM=Tibi%2C+Daniel
Ist gefixt.
